eap-xp2/image.yaml

```diff
@@ -0,0 +1,135 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp2-openjdk11-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform XP 2.0 OpenShift container image."
+version: "1.0"
```

Comment:
the version of the image should match the version of the EAP XP release to be consistent with our versioning scheme.

E.g current EAP 7 image is at 7.3.4: https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/5a6b7f76e510c7abaac78123f358f19a2e28133d/image.yaml#L5

Reply:
+1, I had it this way originally, but when we switched over to the new subdir, I thought this should match the version of XP1, which we started at :1.0, but is a bit ambigious. Will update this to be :2.0 again.